lib/alexa.js

```diff
@@ -367,6 +367,9 @@ function ResponseBuilder(self) {
 
                 responseObject.response.directives = [audioPlayerDirective];
                 return this;
+            },
+            'setResponse': function () {
```

Comment:
Would now be a good time to add some dialog directive functionality to `ResponseBuilder`?  Or are you guys no longer adding features to `this.response` in favour of the more standard ways of sending response (i.e. `:ask` and `:tell`)?